indexing-service/src/main/java/io/druid/indexing/overlord/ThreadPoolTaskRunner.java

```diff
@@ -335,6 +379,11 @@ public TaskStatus call()
 
       try {
         log.info("Running task: %s", task.getId());
+        TaskRunnerUtils.notifyLocationChanged(
```

Comment:
@drcrallen actually, I am gonna leave this the way it is, as it's happening in the task running thread so this is an appropriate time to notify that a location exists. It's ok that this is before the task is fully started up- the location is allowed to be notified before then.